plugin.video.formula1/resources/settings.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<settings>
+    <category label="30001">
+        <setting label="30010" type="lsep"/>
+        <setting id="list.size" type="labelenum" label="30011" values="5|10|15|20|25" default="15"/>
+        <setting label="30020" type="lsep"/>
+        <setting id="video.format" type="labelenum" label="30021" values="360p|720p|1080p|HLS (Adaptive)" default="HLS (Adaptive)"/>
```

Comment:
Because ffmpeg is builtin and inoutstream is an add-on. If you search for inoutstream in repo-plugins you'll find loads of examples. Even the readme of the add-on/wiki has an hls example. It is fine for me, this is not a request but rather a suggestion.